src/components/card/card-body.ts

```diff
@@ -97,80 +87,93 @@ export class CardBody {
   }
 
   private readonly processNode = (node: HTMLElement): HTMLElement => {
-    const elementFromNode: HTMLElement = node;
-    if (elementFromNode.tagName?.toLowerCase() === 'a') {
-      const url = elementFromNode.getAttribute('href') ?? '';
-      return DomBuilder.getInstance().build(
-        {
-          type: 'a',
-          events: {
-            click: (e: MouseEvent) => {
-              if (this.props.onLinkClick !== undefined) {
-                this.props.onLinkClick(url, e);
-              }
-            },
-            auxclick: (e: MouseEvent) => {
-              if (this.props.onLinkClick !== undefined) {
-                this.props.onLinkClick(url, e);
-              }
+    let elementFromNode: HTMLElement = node;
```

Comment:
nit:  this function is too big, let's consider putting logic into functions like:
```
processNode(){
 if(text.node)
   animateTextNode()
  else if(linkNode)
    linkNode()

}
```

Reply:
Next plan is to move it to a helper function